src/ui/RealmBrowser/Content/Table/types/DictionaryCell.tsx

```diff
@@ -0,0 +1,68 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2021 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import React from 'react';
+import { Badge } from 'reactstrap';
+import Realm from 'realm';
+import { asSafeJsonString } from '../../../../../utils/json';
+
+import * as primitives from '../../../primitives';
+
+// TODO: Get declaration from Realm
```

Comment:
Let's remember this once an official release with dictionaries is out 👍

Reply:
Yeah, they're not in realm-js just yet.